src/test/java/woowaapplication/pair/exchange/currency/application/CurrencyServiceTest.java

```diff
@@ -0,0 +1,61 @@
+package woowaapplication.pair.exchange.currency.application;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import woowaapplication.pair.exchange.currency.application.dto.CurrencyRequestDto;
+import woowaapplication.pair.exchange.currency.application.dto.CurrencyResponseDto;
+import woowaapplication.pair.exchange.currency.domain.Nations;
+import woowaapplication.pair.exchange.currency.fixture.CurrencyFixture;
+
+
+@SpringBootTest
+class CurrencyServiceTest {
+
+    @Autowired
+    private CurrencyService currencyService;
+
+    @Test
+    @DisplayName("하나의 수취국가에 대한 환율 및 환율 변환 결과 가져오기")
+    public void currencyGetOneCurrencyTest (){
+        //given
+        CurrencyRequestDto currencyRequestDto = new CurrencyRequestDto(Nations.USA, Nations.JAPAN,
+            1000.0);
```

Comment:
환율 정보를 1000.0 대신 fixture 데이터를 사용한다면 가독성이 더 좋을 것 같습니다 :)

```suggestion
            일본.get환율());
```